package.json

```diff
@@ -63,6 +61,7 @@
     "request": "^2.81.0"
   },
   "dependencies": {
+    "argon2": "^0.15.0",
```

Comment:
I'm leaning toward using many rounds of `SHA-256`. I would rather stick to a time-tested hash algorithm and increase the number of rounds to slow crackers than to rely on a new, less-tested algorithm.

https://en.wikipedia.org/wiki/SHA-2#Cryptanalysis_and_validation

https://en.wikipedia.org/wiki/Argon2#Cryptanalysis